walkthroughs/week-3-server/examples/server-stats/src/main/webapp/script.js

```diff
@@ -19,7 +19,8 @@ function getServerStats() {
   fetch('/server-stats').then(response => response.json()).then((stats) => {
     // stats is an object, not a string, so we have to
     // reference its fields to create HTML content
-
+	console.log("hehe I'm in the console");
+    console.log(stats);
```

Comment:
this should not just log stats, include string to indicate that the object is stats.